python2verilog/ir/context.py

```diff
@@ -298,9 +306,9 @@ def get_strs(variables: Sequence[Var]):
 
         assert isinstance(name, str)
         variables = [
-            *list(get_strs(self._global_vars)),
-            *list(get_strs(self._input_vars)),
-            *list(get_strs(self._output_vars)),
+            *list(get_strs(self.global_vars)),
+            *list(get_strs(self.input_vars)),
+            *list(get_strs(self.output_vars)),
```

Comment:
I would double check this logic to make sure you're accessing the array of variables instead of the function